Index: rccl-rocm-5.7.1/CMakeLists.txt
===================================================================
--- rccl-rocm-5.7.1.orig/CMakeLists.txt
+++ rccl-rocm-5.7.1/CMakeLists.txt
@@ -13,7 +13,7 @@
 option(BUILD_ALLREDUCE_ONLY                    "AllReduce(sum,float) kernel only"              OFF)
 option(BUILD_BFD                               "Enable custom backtrace (if bfd.h exists)"     ON)
 option(BUILD_FILE_REORG_BACKWARD_COMPATIBILITY "File/folder reorg with backward compatibility" ON)
-option(BUILD_LOCAL_GPU_TARGET_ONLY             "Build only for GPUs detected on this machine"  OFF)
+option(BUILD_LOCAL_GPU_TARGET_ONLY             "Build only for GPUs detected on this machine"  OFF)
 option(BUILD_SHARED_LIBS                       "Build as shared library"                       ON)
 option(BUILD_TESTS                             "Build unit test programs"                      OFF)
 option(COLLTRACE                               "Collective Trace Option"                       ON)
@@ -555,7 +555,6 @@
 endif()
 
 ## Set RCCL compile options
-target_compile_options(rccl PRIVATE -parallel-jobs=12)
+target_compile_options(rccl PRIVATE -parallel-jobs=12)
 target_compile_options(rccl PRIVATE -Wno-format-nonliteral)
 target_compile_options(rccl PRIVATE -fgpu-rdc)               # Generate relocatable device code (required for extern __shared__)
 target_compile_options(rccl PRIVATE -fvisibility=hidden)     # Set symbol visibility to hidden by default
@@ -593,7 +592,6 @@
 
 ## Set RCCL link options
 target_link_options(rccl PRIVATE -fgpu-rdc)               # Required when linking relocatable device code
-target_link_options(rccl PRIVATE -parallel-jobs=16)       # Use multiple threads to link
+target_link_options(rccl PRIVATE -parallel-jobs=16)       # Use multiple threads to link
 if(BUILD_ADDRESS_SANITIZER)
   target_link_options(rccl PRIVATE -fuse-ld=lld)
 endif()
